test/ServerInvoiceTest.php

```diff
@@ -319,6 +319,8 @@ public function itShouldCreateInvoiceWhenSubscribedOnBasePlanAndConsumableAddon(
         self::assertSame($this->account->getAccountId(), $subscriptionBase->getAccountId());
         self::assertSame('sports-monthly', $subscriptionBase->getPlanName());
 
+        $this->clock->addDays(1); //otherwise gas-monthly creates extra invoice for 1 day
```

Comment:
In this test only 2 invoices are expected. I got an [error in CircleCI](https://circleci.com/gh/killbill/killbill-client-php/111) for this PR that 3 invoices were created:
![image](https://user-images.githubusercontent.com/1675033/74030628-2e6e1d80-49b8-11ea-863d-5b7f23f82ca1.png)

This is what happened:
- First invoice: (today) $0 item for sports-monthly trial
- Second invoice: (after 29 days) $0 item for 1 day of gas-monthly <----redundant
- Third: (after 30 days) $500 item for sports-monthly and $0 item for gas-monthly

If I add this 1 day delay the second invoice disappears. 
I also reproduced this in master. Killbill version: 22.1 and 22.2.

@pierre do you have an idea why second invoice gets added?

Reply:
I'm confused about the issue 😕 Is it still happening for you? I just ran the test locally without the extra day, and it passes.

Reply:
I have commented this line out and updated killbill to 0.22.6. Let's see if it passes.

Reply:
Fixed! So this PR can be merged.
P.S I suggest that you add a cron job for builds https://circleci.com/docs/2.0/triggers/

Reply:
Awesome! Thanks!